src/symbol.cr

```diff
@@ -14,10 +14,19 @@
 struct Symbol
   include Comparable(Symbol)
 
+  # Compares symbol with other_symbol based on `String#<=>` method. Returns -1, 0
```

Comment:
The docs will render the argument name as `other`, talking about `other_symbol` could be slightly confusing, I'd say unify that by changing either to the other and it's good :)